hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/protocolPB/OzoneManagerProtocolClientSideTranslatorPB.java

```diff
@@ -277,6 +269,22 @@ private RetryAction getRetryAction(RetryAction fallbackAction,
     return proxy;
   }
 
+  /**
+   * Check if exception is a NotLeaderException
+   * @return NotLeaderException.
+   */
+  private NotLeaderException getNotLeaderException(Exception exception) {
+    Throwable cause = exception.getCause();
+    if (cause instanceof RemoteException) {
```

Comment:
Do we need to check cause!= null?

Reply:
I dont think cause can be null. Let me verify.

Reply:
Updated to check that cause is not null.